Havit.Blazor.Components.Web.Bootstrap.Documentation/Pages/Components/HxCalendarDoc/HxCalendar_TimeProvider.CodeSnippet.razor

```diff
@@ -0,0 +1,24 @@
+﻿<!-- Parameter -->
+<HxCalendar TimeProvider="usersTimeProvider" />
+
+<!-- Settings -->
+<HxCalendar Settings="UsersCalendarSettings" />
+
+<!-- Defaults -->
+<HxCalendar />
+@code {
+
+	static TimeProvider usersTimeProvider = new ZonedTimeProvider(TimeZoneInfo.FindSystemTimeZoneById("UTC-11"));
+
+	// OR
+
+	public static CalendarSettings UsersCalendarSettings => new()
+		{
+			TimeProvider = usersTimeProvider,
+		};
+
+	// OR
+
+	HxCalendar.Defaults.TimeProvider = usersTimeProvider
```

Comment:
This should be somewhere in `Program.cs` or somewhere where you maintain you configuration of the defaults, not the `@code` section of a component. I would suggest to add at least a comment here:
```
// OR

// Program.cs or somewhere where you maintain the defaults
HxCalendar.Defaults.TimeProvider = usersTimeProvider;
```